three_regex.js

```diff
@@ -0,0 +1,18 @@
+// Напишіть regex, котрий знайде cat та can, але не знайде car
+var text = 'cat car can';
+var regex = /ca[t|n]/g;
+var matches = text.match(regex);
+console.log(matches);
+
+// Напишіть regex, котрий знайде ВСІ слова котрі починаются на h
+var text = 'Hello people of 1974. I come from the future. In 2014 we have laser guns, hover boards and live on the moon!';
+var regex = /\bh\w*/gi;
```

Comment:
You use \w*, which matches any letter, number, or underscore, but you could use the more precise \w+, which only matches one or more letters, which will avoid matching numbers. It is also possible to replace \b with a simple word beginning ^, since we are looking for words that start with h.

var regex = /^h\w+/gi;

Reply:
You suggesting replace word boundry \b  with string boundry ^ - which will bring expression will found only 1st match.
\w* difference from \w+ - is in amount of characters expected after 'letter we are looking for' - if we replace * with + - it will miss if it will be only 'h' without any letters after. Maybe it's a case we need to thing about.